animation_nodes/base_types/node_tree.py

```diff
@@ -16,8 +16,8 @@ class LastTreeExecutionInfo(bpy.types.PropertyGroup):
 
```

Comment:
I tried `from ..__init__ import bl_info ` so that the versions can be updated automatically, but it has caused an error.
Do you have any idea why?

Reply:
Try without `__init__`.

Reply:
Thanks, i implemented it in 09dc669979f66237a2ed54e5f13430e5118d1a21.